content/en/docs/tasks/administer-cluster/nodelocaldns.md

```diff
@@ -8,36 +8,49 @@ content_type: task
 ---
  
 <!-- overview -->
-{{< feature-state for_k8s_version="v1.18" state="stable" >}}
-This page provides an overview of NodeLocal DNSCache feature in Kubernetes.
 
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
 
+This page provides an overview of NodeLocal DNSCache feature in Kubernetes.
 
 ## {{% heading "prerequisites" %}}
 
-
- {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
-
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
nit (wrong in original): either this page shouldn't use the version-check shortcode, or the front matter should declare a minimum version.

(You can use that bare shortcode if it follows some freeform text that explains custom version requirements, but that doesn't apply here).